libiocage: activate and deactivate command

gronke opened the pull request:
- Use libiocage in `activate` and `deactivate` commands

william-gr commented on iocage_cli/activate.py:
```diff
@@ -21,22 +22,49 @@
+
+    if iocage_pool is None:
+        logger.error(f"ZFS pool '{zpool}' not found")
+        exit(1)
```
We could do:

```
try:
    iocage_pool = zfs.get(zpool)
except Exception:
    logger.error(f"ZFS pool '{zpool}' not found")
    exit(1)
```

william-gr commented on iocage_cli/activate.py:
```diff
@@ -21,22 +22,49 @@
+            mountpoint=mountpoint
+        )
+        logger.log(f"ZFS pool '{zpool}' activated")
+    except iocage.errors.IocageException:
```
Shouldnt we print something to the user in case that happens? Does that exception have an error message?

gronke replied:
The default logger always prints `error` and `warn` levels, so that each such error will already have printed to the users screen using the CLI. 

When an `iocage.error.IocageException` occurs, we know that the user was notified already.

gronke committed:
activate/deactivate command use libiocage

igalic commented on iocage_cli/activate.py:
```diff
@@ -21,22 +22,45 @@
 
 __rootcmd__ = True
 
 
```
how come pep8's not complaining about so many spaces here 😁

gronke replied:
It would in case you choose any other number of newlines.

igalic commented on iocage_cli/deactivate.py:
```diff
@@ -0,0 +1,62 @@
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
+# IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Deactivate ZFS pools for iocage with the CLI."""
```
i didn't even know we had that!

igalic reviewed (approved):
👀

gronke committed:
address suggested improvements from @william-gr

https://github.com/iocage/iocage/pull/633/files/0cf4edf2e6251ccb01c7ea729d63e0b1fbadf5d9#r223096783
